conceptnet5/vectors/evaluation/bias.py

```diff
@@ -0,0 +1,197 @@
+import numpy as np
+import pandas as pd
+import scipy
+
+from conceptnet5.vectors import standardized_uri, normalize_vec
+from conceptnet5.vectors.transforms import (
+    l2_normalize_rows, subtract_mean_vector
+)
+from conceptnet5.vectors.debias import (
+    FEMALE_WORDS, MALE_WORDS, PEOPLE_BY_BELIEF, PEOPLE_BY_ETHNICITY,
+    CULTURE_PREJUDICES, get_category_axis, get_vocabulary_vectors
```

Comment:
It's used at the bottom, in `de_bias_frame`. It's not currently used separately from `SEX_PREJUDICES`, I suppose.